### PR TITLE
create Lovelace class extending SapphireClient with custom constructor, login, and destroy methods

### PR DESCRIPTION
**Pull request by Ca3dyn**
#17 pull request for this issue

### CODE REVIEW

File: src/index.ts
Comment (leavism) [inline src/index.ts:1]: Thanks for separating the options out, makes it more legible.

File: src/lib/LovelaceClient.ts
Comment (leavism) [inline src/lib/LovelaceClient.ts:1]: Please add JSDocs to this class. It should be pretty barebones but at least get it started.
https://jsdoc.app/about-getting-started

**Review by leavism — CHANGES_REQUESTED** [review]
Add JSDocs, get ESLint and prettier working, and you should be good to go.

**Comment by Ca3dyn** [comment]
#18 pull request for JSDocs

File: .vscode/settings.json
Comment (leavism) [inline .vscode/settings.json:1]: Please double check what you're committing before you push them.

File: src/lib/LovelaceClient.ts
Comment (leavism) [inline src/lib/LovelaceClient.ts:1]: New JSDocs are good enough. I'm going to change some verbiage before I merge ("logs the _user_ into Discord" when it should be bot).

**Review by leavism — COMMENTED** [review]
@Ca3dyn you probably want to get out of the habit of doing `git add .` or `git add -A` because you'll accidentally add files you didn't mean to, like you did this time.

**Review by leavism — APPROVED** [review]
Made final changes, ready to merge!

**Pull request merged**